apps/client/src/pages/time-table/components/time-table-item/time-table-item.css.ts

```diff
@@ -0,0 +1,87 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+import { themeVars } from '@confeti/design-system/styles';
+
+export const itemsWrapper = recipe({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'absolute',
+    top: 'calc( 0.7rem + var(--top) )',
+    left: 'calc( 3.1rem + ((100% - 3.1rem) / var(--stage-count) * var(--stage-order)))',
+    height: 'var(--diff)',
+    width: 'calc((100% - 3.1rem) / var(--stage-count))',
+    padding: '0.8rem 0rem',
+    borderRadius: '2px',
+    zIndex: '3',
+    cursor: 'pointer',
+  },
```

Comment:
```suggestion
transition: 'background-color 0.3s ease, color 0.3s ease',
  },
```

transition 효과 적용 부탁드릴게요! 초는 디자이너분께 의견 여쭤보셔도 됩니다